docs/en/reference/advanced-configuration.rst

```diff
@@ -28,7 +29,7 @@ steps of configuration.
 
     $config = new Configuration;
     $config->setMetadataCache($metadataCache);
-    $driverImpl = ORMSetup::createDefaultAnnotationDriver('/path/to/lib/MyProject/Entities');
+    $driverImpl = new AttributeDriver('/path/to/lib/MyProject/Entities');
```

Comment:
We should recommend using the attribute driver on 2.x already, maybe with a little not that it requires PHP 8.

Reply:
Yes, while looking into this I noticed there are a lot more things to do on 2.x regarding the attribute and yaml driver deprecations.